source/app/web/statics/app.placeholder.js

```diff
@@ -565,7 +565,7 @@
         ...(set.plugins.enabled.nightscout
           ? ({
             nightscout: {
-              url: options["nightscout.url"] != null && options["nightscout.url"] != "https://example.herokuapp.com" ? options["nightscout.url"] : "https://testapp.herokuapp.com/",
+              url: options["nightscout.url"] !== null && options["nightscout.url"] != "https://example.herokuapp.com" ? options["nightscout.url"] : "https://testapp.herokuapp.com/",
```

Comment:
By doing a strict not-equals, this will resolve to `true` if `options["nightscout.url"]` is undefined.
```javascript
const foo = {}
foo.bar != null // false
foo.bar !== null // true
```